.github/workflows/main.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       - name: go mod tidy
         run: make tidy
       - name: Build
-        run: make mpi-operator.v1 mpi-operator.v2 kubectl-delivery
+        run: make mpi-operator.v2 kubectl-delivery
```

Comment:
the v2 operator doesn't need kubectl-delivery, so it can be removed as well.

Reply:
Ok.
I will create a PR to remove that.